index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="header">
+    <div class="up"><img src="icons/Dark.png"></div>
+    <div class="logo">My Logo</div>
+    <div class="union"><img src="icons/Union.png"></div>
+  </div> 
+  <div class="headline">
+    <div class="text">Hey There. <br> I'm Alex.</div>
+    <div class="text2">I'm a Software Developer</div>
+    <div class="text3">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</div>
+    <div class="icon1"><img src="icons/icon1.png"></div>
+    <div class="icon2"><img src="icons/icon2.png"></div>
+    <div class="icon3"><img src="icons/icon3.png"></div>
+    <div class="icon4"><img src="icons/icon4.png"></div>
+    <div class="icon5"><img src="icons/icon5.png"></div>
+    <div class="illustration"><img src="illustration.png"></div>
```

Comment:
Nice one adding the decorative image at the bottom of the page💪. However, please note that since this image is decorative, it should be added as a background image and not as an img element. You can check this [link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info on why this is convention. Please update this element to be a background image and not an img tag